deploy/release-config/z_original-archive/dev-test-demo-load/admin.txt

```diff
@@ -0,0 +1,24 @@
+-ALLOWED_WORDS $(SchoolPinWordSet)
```

Comment:
@GuyHarwood - why are there a bunch of files added to the `/z_original-archive/` folder?

Reply:
these are the original ones, copied verbatim from the release pipeline definition.

They are to be removed before the PR is closed.  Next steps...

1. finalise reviews
2. apply settings
3. test release
4. dispose of original files in archive
5. sign off
5. close and merge PR

Reply:
Okay, as this is a special case, we can leave them in the PR for now.